src/components/Modal/index.js

```diff
@@ -45,7 +50,11 @@ const Modal = ({
       />
     </div>
 
-    <div className={styles.body} data-testid={getComponentTestId('ModalBody', name || title)}>
+    <div
```

Comment:
How you have broken the div element in many lines and you have only a short and single child value you could utilize the children prop instead.

```suggestion
    <div
      className={styles.body}
      style={contentStyle}
      data-testid={getComponentTestId('ModalBody', name || title)}
      children={children}
    />
```